typewise_alert.test.py

```diff
@@ -5,7 +5,11 @@
 class TypewiseTest(unittest.TestCase):
   def test_infers_breach_as_per_limits(self):
     self.assertTrue(typewise_alert.infer_breach(20, 50, 100) == 'TOO_LOW')
-
+    self.assertEqual(typewise_alert.infer_breach(150, 50, 100), 'TOO_HIGH')
```

Comment:
Test cases for covering checkAndAlert() is missing.
Please add test cases for complete coverage...